packages/connectivity_plus/connectivity_plus/example/integration_test/connectivity_plus_test.dart

```diff
@@ -22,5 +24,14 @@ void main() {
       final result = await _connectivity.checkConnectivity();
       expect(result, isNotNull);
     });
+
+    testWidgets('test wifi or cellular', (WidgetTester tester) async {
+      final result = await _connectivity.checkConnectivity();
+
+      ConnectivityResult _connectionStatus = ConnectivityResult.none;
+      _connectionStatus = result;
+
+      expect(_connectionStatus, equals(_connectionStatusExpected));
```

Comment:
This could simply be `expect(result, ConnectivityResult.wifi)`;

Reply:
Got ~